src/views/Notification/components/NotificationForm/index.jsx

```diff
@@ -57,7 +56,7 @@ const DistanceSearch = ({ store }) => {
  * @param {function(): void} props.onCancelButtonClick - Callback that gets called when the user
  *     clicks the cancel button
  * @param {function(): void} props.onNotifyButtonClick - Callback that receives the edited
- *     parcel when the user clicks the "Notify" button
+ *     parcel when the user clicks the "Notify" button.
```

Comment:
The callback doesn't actually receive a parcel anymore. This documentation is deprecated. Let's just go with "Callback that gets called when the user clicks the notify button"

Reply:
Done